feature-libs/product-configurator/rulebased/components/attribute/types/numeric-input-field/configurator-attribute-numeric-input-field.component.ts

```diff
@@ -72,29 +87,21 @@ export class ConfiguratorAttributeNumericInputFieldComponent
     if (this.attribute.userInput) {
       this.attributeInputForm.setValue(this.attribute.userInput);
     }
-  }
-
-  /**
-   * Hit when user input was changed
-   */
-  onChange(): void {
-    const event: ConfigFormUpdateEvent = this.createEventFromInput();
 
-    if (!this.attributeInputForm.invalid) {
-      this.inputChange.emit(event);
-    }
+    this.sub = this.attributeInputForm.valueChanges
+      .pipe(
+        debounce(() =>
+          timer(
+            this.config?.productConfigurator?.debounceTime?.input ??
+              this.FALLBACK_DEBOUNCE_TIME
+          )
+        )
+      )
+      .subscribe(() => this.onChange());
   }
 
-  protected createEventFromInput(): ConfigFormUpdateEvent {
```

Comment:
Remove this function will cause breaking change. So, we have to deprecate it.

Reply:
Picked!

Reply:
Done!